### ISSUE TITLE
Incomplete BSD License File

### ISSUE THREAD

**Issue by WPCTW** [CocoaLumberjack/CocoaLumberjack#936]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/CocoaLumberjack/CocoaLumberjack/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](https://github.com/CocoaLumberjack/CocoaLumberjack#documentation) or [CocoaDocs](cocoadocs.org/docsets/CocoaLumberjack)
* [ ] I have searched for a similar issue in the [project](https://github.com/CocoaLumberjack/CocoaLumberjack/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / osx / tvos / watchos
 Platform Version        | e.g. 8.0
 CocoaLumberjack Version | e.g. 2.3.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 7.3
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

Please check the License file, because the License does not include full text of BSD license.
If this project applies to BSD license, it normally would be BSD 2-Clause license or BSD 3-Clause license.
Please refer to the following links: 
1. BSD 2-Clause License (https://opensource.org/licenses/BSD-2-Clause)
2. BSD 3-Clause License (https://opensource.org/licenses/BSD-3-Clause)

**Comment by pombredanne** [comment]
This is a rare, franken license that consiste of the first and 3rd clause of the BSD 3 clause 
I track this in scancode as the "mentalis" license.

https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/licenses/mentalis.LICENSE

It is misleading to name this a BSD license as this is a rare bird indeed.

**Comment by bpoplauschi** [comment]
@WPCTW @pombredanne thanks for your input guys. I have created #942, going to a full BSD 3-Clause License. I'm waiting for a review from @robbiehanson

**Comment by pombredanne** [comment]
@bpoplauschi thank you for the offer, but I would be grossly under qualified to be a maintainer on this codebase, so I will have to pass.

**Comment by bpoplauschi** [comment]
@pombredanne that ok, thanks for responding :) good luck with your work
